data/json/npcs/missiondef.json

```diff
@@ -732,5 +732,15 @@
       "success_lie": "OK, then hand it over.",
       "failure": "Shit happens."
     }
+  },
+  {
+    "id": "MISSION_INFECTED_START_FIND_ANTIBIOTICS",
+    "type": "mission_definition",
+    "name": "Find Antibiotics Before You Die!",
```

Comment:
I'd decapitalize the name.

Reply:
This is following the pattern that exists already, let's have an issue to discuss that if we want to change the pattern.